content/en/_index.md

```diff
@@ -155,13 +189,30 @@ public static void listInfoTypes() throws IOException {
     ListInfoTypesResponse response = dlpClient.listInfoTypes(listInfoTypesRequest);
 
     // Parse the response and process the results
-    System.out.println("Infotypes found:");
+    System.out.println("Info types found:");
     for (InfoTypeDescription infoTypeDescription : response.getInfoTypesList()) {
       System.out.println("Name : " + infoTypeDescription.getName());
       System.out.println("Display name : " + infoTypeDescription.getDisplayName());
     }
   }
-}
+}  
+{{< /tab >}}
+{{< tab header="Ruby" >}}
+# Lists the notifications configured for the given bucket.
+def list_bucket_notifications bucket_name:
+  # The ID of your GCS bucket
+  # bucket_name = "your-unique-bucket-name"
+  # Arrange by creating client and request hashes if any
+  storage = Google::Cloud::Storage.new
+
+  # Act by calling the respective action
+  bucket  = storage.bucket bucket_name
+
+  # Assert by parsing the response and processing the result
+  bucket.notifications.each do |notification|
+    puts "Notification ID: #{notification.id}"
+  end
+end
```

Comment:
This snippet is different than the Java one - do we think they should be consistent or does it not matter? 

cc @grayside

Reply:
I think we want to use the same use case/product for each language. If someone is reading the guide using multiple language to refine their understanding of the rule, similarity helps a lot.

Here's an example of what I'm thinking about: an engineer new to sample writing has been asked to contribute a Ruby sample, but they are more familiar with Java and Node.js. Similarity of all languages means they can tab to a more familiar language to understand the rule, then tab back to Ruby to understand how they'll apply it.

I don't have strong thoughts about needing to use DLP for each rule.

Reply:
I don't have any preference to using DLP either. I'm open to updating to something else, I just think we should decide what.

Reply:
Sure will update the samples here 👍